codegen/src/main/scala/org/apache/pekko/grpc/gen/scaladsl/Serializer.scala

```diff
@@ -16,7 +16,7 @@ package org.apache.pekko.grpc.gen.scaladsl
 import com.google.protobuf.Descriptors.{ Descriptor, MethodDescriptor }
 import scalapb.compiler.DescriptorImplicits
 
-case class Serializer(name: String, init: String)
+case class Serializer(name: String, scalaType: String, init: String)
```

Comment:
This change is necessary in order to provide an explicit type to `implicit val` definitions (which are mandatory in Scala 3). Note that this isn't really a public class since its part of pekko-grpc's codegen, in other words no one should be depending on it.